### PR TITLE
REF: Handoff Component

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/43a35ae05tanu4r025qauv7y8c

[download apk](http://filestorage.bluewallet.io:1488/refhandoff-1611028374.apk)

File: blue_modules/handoff.ios.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import Handoff from 'react-native-handoff';
+import { BlueStorageContext } from './storage-context';
+import PropTypes from 'prop-types';
+
+const HandoffComponent = props => {
+  const { isHandOffUseEnabled } = useContext(BlueStorageContext);
+
+  return isHandOffUseEnabled && props && props.url ? <Handoff {...props} /> : null;
+};
+HandoffComponent.STORAGE_KEY = 'HandOff';
```
Comment (limpbrains) [inline blue_modules/handoff.ios.js:11]: `HandoffComponent.STORAGE_KEY` is the same as `AppStorage.HANDOFF_STORAGE_KEY` but it's never used.

**Review by limpbrains — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/u8q71m94krakwtd1j224uy5dpm

[download apk](http://filestorage.bluewallet.io:1488/refhandoff-1611059198.apk)

**Review by limpbrains — APPROVED** [review]

File: blue_modules/handoff.ios.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React, { useContext } from 'react';
+import Handoff from 'react-native-handoff';
```
Comment (Overtorment) [inline blue_modules/handoff.ios.js:2]: feels like this file should be in `components/`

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: blue_modules/handoff.ios.js (hunk 2 above)
Comment (marcosrdz) [reply blue_modules/handoff.ios.js]: done

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/m1af7wcxw4906yuk16jxznafk8

[download apk](http://filestorage.bluewallet.io:1488/refhandoff-1611334906.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/nh30dm0zutdn64th5463hudnpm

[download apk](http://filestorage.bluewallet.io:1488/refhandoff-1611521086.apk)

**Pull request merged**